consensus/mempool_test.go

```diff
@@ -108,7 +108,8 @@ func TestTxConcurrentWithCommit(t *testing.T) {
 	for nTxs := 0; nTxs < NTxs; {
```

Comment:
should we make nTxs int64 like `nTxs := int64(0)`

Reply:
I could, but then I would have to change types here as well:
```
	NTxs := 10000
	go deliverTxsRange(cs, 0, NTxs)
```

simpler to cast here once